README.md

```diff
@@ -11,28 +11,47 @@ gen3 integration tests - run by https://jenkins.planx-pla.net/ via a `Jenkinsfil
 docker run -d -p 4444:4444 --name=selenium --rm -v /dev/shm:/dev/shm selenium/standalone-chrome
 ```
 
-### Test with dev environment
+### Test with dev environment (ssh backend access)
 
-Run a test locally against a dev environment like this:
+Run a test locally against a dev environment that you can ssh to and run gen3 commands like this:
 
 ```
-# basic run - some tests require more setup than this
 RUNNING_LOCAL=true NAMESPACE=yourDevNamespace TEST_DATA_PATH=./testData npm test -- --verbose --grep '@dataClientCLI|@reqGoogle' --invert --reporter mocha-multi suites/.../myTest.js
 ```
 
-### Test an arbirtray remote hostname
+### Test an arbitrary gen3 commons
 
-This currently only works for tests that do not require
-an access token (like interactive tests).
-It's on our TODO list to support tokens in environment variables,
-`secrets.json` configuration, or fetching with an API key.
+Some of our test suites require access to the commons backend, but most of our tests can run against an arbitrary commons given an access token that can interact with the gen3 api.
 
-```
-# skip program/project creation at startup for now ...
+The `getAccessToken.sh` script is a little helper that can retrieve an access token given an API key.
+
+* First - generate some test data (if necessary for the test) with the appropriate dictionary (see below)
+* Next - generate an access token (it will expire in 20 minutes) using an API key from the target commons
 
+```
 mkdir -p output
+export GEN3_TOKEN_MAIN=$(bash ./getAccessToken.sh ~/Secrets/gen3/qa-brain.planx-pla.net/credentials.json)
```

Comment:
Should we consider some err check here? e.g.,
```
output=$(bash ./getAccessToken.sh ~/Secrets/gen3/qa-brain.planx-pla.net/credentials.json)
RC=$?
if [ $RC -ne 0 ]; then
  echo "something went wrong while trying to obtain access token."
  exit 1
fi

export GEN3_TOKEN_MAIN=$output
```

Approving anyway as it's not strictly necessary.